src/components/overview/chart/index.tsx

```diff
@@ -0,0 +1,91 @@
+import CustomizedDot from '../customizedDot';
+import LinearGradients from '../linearGradients';
+import { colors } from '../../../theme';
+import { useState } from 'react';
+import {
+  Area,
+  AreaChart,
+  CartesianGrid,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import { data, gradients, yDomain, yTicks } from './helper';
+import { tick, xTickPadding } from './styled';
+interface IEntry {
+  today: number;
+  hours: string;
+}
+const Chart = () => {
+  const currentDate = new Date();
+  const currentHour = currentDate.getHours();
+  const [isTodayHovered, setIsTodayHovered] = useState(false);
+  const [isYesterdayHovered, setIsYesterdayHovered] = useState(false);
+
+  const calculateTodayValue = (entry: IEntry, currentHour: number): number | null => {
+    return entry.today && parseInt(entry.hours) <= currentHour ? entry.today : null;
+  };
+
+  const currentData = data.map((entry) => ({
+    ...entry,
+    today: calculateTodayValue(entry, currentHour),
+  }));
+
+  return (
+    <ResponsiveContainer width='100%' height={380}>
+      <AreaChart data={currentData} margin={{ top: 20, right: 30, left: 5 }}>
+        <defs>
+          <LinearGradients gradients={gradients} />
+        </defs>
+        <CartesianGrid vertical={false} stroke={colors.graphDivider} />
+        <XAxis
+          dataKey='hours'
```

Comment:
Let's decrease an amount of rows in this component. Let's move all of this props to the helper `const XAxisConfig = {dataKey: 'hours'...}` and then use `<XAxis {...XAxisConfig} />`

Reply:
fix: X & Y props moved in helper